impl/asm/src/main/java/org/wildfly/transformer/asm/TransformerImpl.java

```diff
@@ -294,12 +338,13 @@ private void generateReflectionHandlingModelCode(String handlingClassPackage) {
                                 out.flush();
                                 byte[] bataviaReflectionModel = out.toByteArray();        
                                 ClassReader bataviaReflectionModelClassReader = new ClassReader(bataviaReflectionModel);
-                                final ClassWriter bataviaReflectionModelClassWriter = new ClassWriter(bataviaReflectionModelClassReader, 0);
+                                final ClassWriter bataviaReflectionModelClassWriter = new ClassWriter(bataviaReflectionModelClassReader, ClassWriter.COMPUTE_FRAMES);
                                 bataviaReflectionModelClassReader.accept(new ClassVisitor(useASM7 ? Opcodes.ASM7 : Opcodes.ASM6, bataviaReflectionModelClassWriter) {
```

Comment:
@ropalka note that the COMPUTE_FRAMES option is required due to the modification of the ReflectionModel bytecode.

Reply:
Yes, I know it needs to be recalculated @scottmarlow